py4j-java/src/main/java/py4j/CallbackClient.java

```diff
@@ -90,6 +90,7 @@ public class CallbackClient implements Py4JPythonClient {
 	protected final boolean enableMemoryManagement;
 
 	protected final int readTimeout;
+	protected final int createSocketTimeout;
```

Comment:
should this be `connectTimeout`?

```suggestion
	protected final int connectTimeout;
```

Reply:
+1 for this renaming